packages/web/src/scss/components/SplitButton/index.html

```diff
@@ -0,0 +1,671 @@
+{{#> web/layout/default title="SplitButton" parentPageName="Components" }}
+<section class="UNSTABLE_Section">
+  <div class="Container">
+    <h2 class="docs-Heading">Primary</h2>
```

Comment:
Maybe we can somehow generate this? Similar as we do it in the Button index file?

Reply:
something like `{{#each @root.colors as |color|}}` and generate all section? 🤷